lib/chunked-uploader.js

```diff
@@ -16,21 +71,26 @@ const EventEmitter = require('events').EventEmitter,
 // Private
 // -----------------------------------------------------------------------------
 
-//@TODO: make into an option
+// @TODO: make into an option
```

Comment:
Did you mean to leave this for later? I'm OK with that call, but just checking in case it was an oversight.